sa/storage-authority.go

```diff
@@ -673,6 +674,117 @@ func (ssa *SQLStorageAuthority) FinalizeAuthorization(authz core.Authorization)
 	return
 }
 
+var authorizationTables = []string{
+	"authz",
+	"pendingAuthorizations",
+}
+
+const getAuthorizationIDsMax = 1000
+
+func getAuthorizationIDsByDomain(db *gorp.DbMap, ident string, now time.Time) ([]string, error) {
```

Comment:
Let's move these funcs into their own file. This file is pretty big now and they are nicely logically separate